integration/helpers/stack.go

```diff
@@ -61,21 +61,42 @@ func CreateStack(names ...string) string {
 		`{"name":"%s","description":"CF CLI integration test stack, please delete"}`,
 		name,
 	)
-	session := CF("curl", "-v", "-X", "POST", "/v2/stacks", "-d", requestBody)
+	session := CF("curl", "-v", "-X", "POST", "/v3/stacks", "-d", requestBody)
```

Comment:
This helper is used in an older test but I think it should be ok as it will be skipped on our legacy environment:
https://github.com/cloudfoundry/cli/blob/afa7edfedc4f16251eb549ebd0d5fea30c3a5c25/integration/v6/push/buildpack_test.go#L76